src/components/SelectTokenTextField/styles.ts

```diff
@@ -77,5 +77,8 @@ export const useStyles = () => {
     whiteLabel: css`
       color: ${theme.palette.text.primary};
     `,
+    maxButton: css`
+      margin-left: ${theme.spacing(1)};
```

Comment:
Sorry I realize you weren't given any design for this task so this is subjective, but just so the token selector and max buttons are spaced out a bit more I'd apply a spacing of 2 (8px):
```suggestion
      margin-left: ${theme.spacing(2)};
```